packages/react-native-reanimated/src/NativeReanimated/NativeReanimated.ts

```diff
@@ -78,19 +78,23 @@ See \`https://docs.swmansion.com/react-native-reanimated/docs/guides/troubleshoo
 }
 
 export class NativeReanimated {
-  private InnerNativeModule: NativeReanimatedModule;
+  #nativeWorklets: NativeWorklets;
+  #nativeReanimatedModule: NativeReanimatedModule;
```

Comment:
This is a bit confusing. Why is one of them a module, but the second one is not? 🤔

Reply:
Yeah I scratched my head on it too. Basically `NativeReanimatedModule` is the type of `__reanimatedModuleProxy`, while `NativeWorklets` is the type of the TypeScript class.

I thought of changing
- `NativeReanimatedModule` to `ReanimatedModuleProxy`
or
- `NativeWorklets` to `NativeWorkletsWrapper` but this would require renaming `NativeReanimatedModule` to `NativeReanimatedWrapper`

Generally, it's hard 🤯